packages/console/src/cloud/pages/Main/TenantLandingPage/index.module.scss

```diff
@@ -0,0 +1,66 @@
+@use '@/scss/underscore' as _;
+
+.placeholder {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+
+  .image {
+    > svg {
+      width: 256px;
+      height: 256px;
+    }
+  }
```

Comment:
Why the `.placeholder` and `.image` are defined again inside `.pageContainer` in L39?

Reply:
Seems you have some CSS to clean up